packages/react-conformance/package.json

```diff
@@ -36,7 +36,7 @@
     "doctrine": "^3.0.0",
     "lodash": "^4.17.15",
     "chalk": "^2.4.2",
-    "react-docgen-typescript": "^1.16.6",
+    "react-docgen-typescript": "^2.1.0",
```

Comment:
This change and the typescript peer dep change are because react-docgen-typescript 2 requires typescript 4.3 due to compiler API changes